starter-code/views/celebrities/index.hbs

```diff
@@ -0,0 +1,17 @@
+<h2>Celebrity Home Page</h2>
+
+<nav>
+
+  <a href="/celebrities/new">Make New Celebrity</a>
+
+</nav>
+<ul>
+  {{#each celebrityArray}}
```

Comment:
on your route `/` the celebrities don't show here because you didn't do a find - check line 6 in your index.js

You're rendering the same hbs file twice, once with the celebrities and once without